scripts/build/python/mu2e_helper.py

```diff
@@ -140,29 +140,28 @@ def make_plugins( self, userlibs, exclude_cc = [], cppf = [], pf = [] ):
     #   Make the dictionary and rootmap plugins.
     #
     def make_dict_and_map( self, userlibs=[], pf_dict=[] ):
-        if os.path.exists('classes.h') and os.path.exists('classes_def.xml'):
```

Comment:
What happens if someone (inappropriately) adds a make_dict_and_map stanza to an SConscript file in a directory that does not have classes.h and classes_def.xml files?   Will there be an understandable error message?  Silently do nothing?

Reply:
you get a message like "don't know how to build classes.h required for .."

Reply:
Thanks. Does it stop the build or is it informational only?

Reply:
It is a fatal scons error and the build stops

Reply:
Good.  That's what I hoped.  I will merge.